lerobot/configs/policy/tdmpc.yaml

```diff
@@ -54,7 +54,7 @@ policy:
   discount: 0.9
 
   # Inference.
-  use_mpc: false
+  use_mpc: true
```

Comment:
FYI: technically this is b/c breaking, but I don't think anyone's using TD-MPC yet as it hasn't been announced to be fully functional